README.md

```diff
@@ -143,19 +143,19 @@ You can use node versions 12.x, 13.x, 14.x
 
 ### `pool = new FixedThreadPool(numThreads, filePath, opts)`
 
-`numThreads` (mandatory) Num of threads for this worker pool  
+`numThreads` (mandatory) Num of workers for this worker pool  
 `filePath` (mandatory) Path to a file with a worker implementation  
 `opts` (optional) An object with these properties :
 
-- `errorHandler` - A function that will listen for error event on each worker thread
-- `onlineHandler` - A function that will listen for online event on each worker thread
-- `exitHandler` - A function that will listen for exit event on each worker thread
+- `errorHandler` - A function that will listen for error event on each worker
```

Comment:
This documentation section is about FixedThreadPool so it is better IMO to keep the "thread" words.

Reply:
We should add maybe an equivalent section for Cluster pools

Reply:
In general I was thinking to publish an html site about our API doc generated from JSDoc or something like

Reply:
The documentation update here just match the JSdoc in the code that is shared between the two worker types.

Reply:
mmmmmm yes so we should duplicate the documentation into the readme? What are your thoughts to improve the readme?

Reply:
Would be great also to document our IPool interface

Reply:
Since the implementation is shared, if
### `pool = new FixedThreadPool(numThreads, filePath, opts)` or `pool = new FixedClusterPool(numThreads, filePath, opts)`
share the same documentation, no need to duplicate the section in the README

Reply:
Should be better now yes

Reply:
https://typedoc.org/

Reply:
@Shinigami92  yes I was thinking to something similar, but you can also tackle this later for me

Reply:
I extract this to a new issue